controllers/vrg_kubeobjects.go

```diff
@@ -364,6 +359,23 @@ func (v *VRGInstance) kubeObjectsRecover(s3ProfileName string, objectStore Objec
 		kubeObjectsRequestNameLabelKey, recoverNamePrefix)
 }
 
+func (v *VRGInstance) veleroNamespaceName() string {
+	veleroNamespaceName := VeleroNamespaceNameDefault
+
+	_, ramenConfig, err := ConfigMapGet(v.ctx, v.reconciler.APIReader)
+	if err != nil {
+		v.log.Error(err, "veleroNamespaceName config failed")
+
+		return veleroNamespaceName
+	}
+
+	if ramenConfig.VeleroNamespaceName != "" {
+		veleroNamespaceName = ramenConfig.VeleroNamespaceName
+	}
+
+	return veleroNamespaceName
+}
+
```

Comment:
Looks good to me!  An optimization nit is the config map could be read only once per reconcile and all the config values stored in say the `VRGInstance`.  The only other config fields used by VRG reconciler that come to mind are the `S3Profiles`.